README.md

```diff
@@ -1,27 +1,27 @@
-# kafka-connect-xml-converter
+# Kafka Connect XML converter and transformation plug-ins
 
-A Kafka Connect plugin to make it easier to work with XML data in Kafka Connect pipelines.
+The Kafka Connect XML converter and transformation plug-ins make it easier to work with XML data in Kafka Connect pipelines.
 
 ## Contents
 
 - `com.ibm.eventstreams.kafkaconnect.plugins.xml.XmlConverter`
-    - a Kafka Connect converter for converting to/from XML strings
+  A Kafka Connect converter for converting between Kafka Connect’s internal data format and XML strings.
 - `com.ibm.eventstreams.kafkaconnect.plugins.xml.XmlTransformation`
-    - a Kafka Connect transformation for converting Kafka Connect records to/from XML strings
+  A Single Message Transform (SMT) that takes a Kafka Connect record containing an XML string and transforms it into a structured Connect record.
 - `com.ibm.eventstreams.kafkaconnect.plugins.xml.XmlMQRecordBuilder`
-    - an MQ Source Record builder for parsing MQ messages containing XML strings
+  - An MQ Source Record builder for parsing MQ messages containing XML strings.
```

Comment:
minor formatting inconsistency - the first two are single-line whereas the third is an indented bullet 

<img width="1200" alt="image" src="https://github.com/user-attachments/assets/fd5d6c16-7f3b-49e2-80bb-059c6a2485c3" />

Please make all three consistent